test/user/signup.js

```diff
@@ -105,15 +105,95 @@ describe('Signup API', () => {
       .set('Content-Type', 'application/json')
       .send({
         name: 'Full name',
-        email: 'testtt@test.com',
-        username: 'user',
+        email: 'tester@test.com',
+        username: 'user_test',
         password: 'password'
       })
       .end((err, res) => {
+        // console.log(res);
```

Comment:
I would suggest leaving out  redundant code that has been commented out

Reply:
Thanks for pointing this out. The unused line has been removed.